Gopkg.toml

```diff
@@ -92,3 +92,7 @@ required = ["github.com/golang/protobuf/jsonpb"]
 [[constraint]]
   branch = "master"
   name = "github.com/etcd-io/gofail"
+
+[[constraint]]
```

Comment:
It's better to move the vendor change to another pr.

Reply:
recommand review by commits, 1st comment add vendor, 2nd replace import and remove vendor, 3rd fix some testcase.